modules/vf-graphql-holochain/queries/intent.ts

```diff
@@ -5,20 +5,20 @@
  * @since:   2019-08-31
  */
 
-import { DNAIdMappings } from '../types'
+import { DNAIdMappings, ReadParams } from '../types'
 import { mapZomeFn } from '../connection'
 
 import {
-  Intent,
+  Intent, IntentResponse,
 } from '@valueflows/vf-graphql'
 
 // :TODO: how to inject DNA identifier?
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
-  const readRecord = mapZomeFn(dnaConfig, conductorUri, 'planning', 'intent', 'get_intent')
+  const readRecord = mapZomeFn<ReadParams, IntentResponse>(dnaConfig, conductorUri, 'planning', 'intent', 'get_intent')
 
   return {
     intent: async (root, args): Promise<Intent> => {
-      return (await (await readRecord)({ address: args.id })).intent
+      return (await (await readRecord)(args)).intent
```

Comment:
whats up with this syntax?

Reply:
that's look like an auto-format of a little syntax mistake, but that still worked in practice. could delete the extra (await ) but leave the `readRecord` it wraps